### DISCUSSION TITLE
Have the ability to stop request running in a flow

### DISCUSSION THREAD

**Discussion by bixelbg** [postmanlabs/postman-flows#122] [Ideas]
First time trying the diagram with my previous implementation and it works great, but something i have in mind is maybe add a "Stop button" after clicking start or ability to stop requests in a block flow would help allot, especially when you have many chained requests it could lead to crashing postman multiple times before managing it better. If someone knows a way it would be greatly appreciated if you share it in the comments

**Comment by samay-v** [comment]
Hi @bixelbg, This is in the works right now and will be available in the webapp soon
